client/app/stories/index.jsx

```diff
@@ -16,6 +22,29 @@ storiesOf('Logo', module)
     <Logo />
   ))
 
+storiesOf('ButtonFilled', module)
+.add('View', () => (
+    <ButtonFilled></ButtonFilled>
+  ));
+storiesOf('ButtonFilledM', module)
```

Comment:
Just like the Logo example above, grouping all buttons into a "Button" story would keep things simple to navigate as our component library grows.